grails-app/domain/org/gmod/chado/CvtermRelationship.groovy

```diff
@@ -2,15 +2,15 @@ package org.gmod.chado
 
 class CvtermRelationship {
 
-    Cvterm cvtermBySubjectId
-    Cvterm cvtermByObjectId
-    Cvterm cvtermByTypeId
+    Cvterm subject
+    Cvterm object
+    Cvterm type
```

Comment:
I think that you are going to explicitly map this to "type_id", "subject_id", "object_id" according to the [cvterm_relationship](http://gmod.org/wiki/Chado_Tables#Table:_cvterm_relationship).  

Should be mapped [this way](http://grails.github.io/grails-doc/2.4.x/ref/Database%20Mapping/column.html):

```
static mapping = {
    subject column: "subjectId", nullable false
}
```